opensplice_cmake_module/env_hook/opensplice.sh.in

```diff
@@ -1,9 +1,15 @@
-export OSPL_HOME=@OpenSplice_HOME@
+fi [ -z "$OSPL_HOME" ]; then
+  export OSPL_HOME=@OpenSplice_HOME@
```

Comment:
Ok, I wasn't sure. I can just isolate the config URI which is the only part necessary.